hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacitySchedulerTestUtilities.java

```diff
@@ -68,11 +73,12 @@ public final class CapacitySchedulerTestUtilities {
   private CapacitySchedulerTestUtilities() {
   }
 
-  public static void setQueueHandler(CapacitySchedulerContext cs) {
+  public static CapacitySchedulerQueueManager setQueueHandler(CapacitySchedulerContext cs) {
```

Comment:
Nit: I'm not sure the name describes the method anymore. CreateMockQueueHandler?